lib/diagnostic.js

```diff
@@ -38,19 +49,33 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = {
+  this.name : 'Dunch',
+  this.species : 'CatDog',
+  this.breed : 'unknown',
+  this._vocalization : 'meow woof',
+  this.vocalize : function(){
+    return this._vocalization;
+  }
+}
 
```

Comment:
let pet = {
  name: 'Beau',
  species: 'Dog',
  breed: 'Lab',
  vocalization: 'bark',
  vocalize: function () {
    return this.vocalization;
  }
};